src/components/OpenWith/OpenWithButton.js

```diff
@@ -69,9 +69,8 @@ const OpenWithButton = ({
             position="bottom-center"
         >
             <Button
-                data-attribute-id={id}
                 isDisabled={isDisabled}
-                onClick={onClick}
+                onClick={onClick ? () => onClick(id) : null}
```

Comment:
`noop` instead of `null`?

Reply:
Is there a benefit? The function won't get called by react if it's `null`

Reply:
Should you even render the button if onClick is absent then?
Also, css below isn't prefixed 'integration-icon'

Reply:
yes, empty and error cases don't have a on click